### ISSUE TITLE
Use keyword hints

### ISSUE THREAD

**Issue by ianb** [mozilla/firefox-voice#493]
If an utterance includes "translate" then we should at least _suggest_ translation. Several intents might reasonably use these to add a suggestion.

**Comment by IkkyOdufade** [comment]
Hi, @ianb . Can I work on this? I have been wanting to understand what intents are and I think this would help.

**Comment by LilyLME** [comment]
Hey @ianb, can I have this?

**Comment by ianb** [comment]
I don't think we have a clear implementable design on this. @awallin do you have thoughts?

My one thought is that we'd add some space in the popup where we could add a bit of text (or maybe use the `eduText` that was recently added). When some keyword is detected, in addition to whatever intent is executed, we'd also show a message about something else you could do.

For instance, you say "organize my window". Right now (and even after this ticket is finished) it goes to [some search result](https://www.howtogeek.com/362241/how-to-organize-your-messy-windows-desktop-and-keep-it-that-way/) and has a simple message in the popup:

<img width="401" alt="image" src="https://user-images.githubusercontent.com/44390/76987327-fa6c0d80-6910-11ea-8916-c1aef8c1ac48.png">

We have a "collect tabs" intent, and maybe we'd want to suggest that if you ever said "organize". So I imagine adding some config like this to `tabs.toml`:

```toml
[[tabs.collectAllTabs.suggestion]]
match = "organize"
suggestion = "Try \"collect all tabs\" to organize your tabs"
allIntents = false
```

The idea is that if a phrase contains "organize" then this is a potential suggestion we'd show. It would be potentially a multiline string like `match` elsewhere, but we'd just be doing string matching (or maybe regexp matching) of the utterance (not the kind of matching used for parsing).

`allIntents` would default to false, and when false it would only apply to fallback searches.

If more than one intent matches then we'd pick one, maybe random to start.

As a followup we'd want to rotate between options when there are multiple matches, and limit the number of times we'd show suggestions (both limiting how often one suggestion is shown, and maybe limiting how often any suggestion is shown).

Note this would primarily happen in `intentRunner.js` (to detect the keywords), and some work in `popupController.jsx` and `popupView.jsx` to show the suggestion.

Assigning to @awallin because I'd like to get his feedback before any implementation.
